autogpt_platform/backend/backend/blocks/discord.py

```diff
@@ -31,6 +31,7 @@ def __init__(self):
             id="d3f4g5h6-1i2j-3k4l-5m6n-7o8p9q0r1s2t",  # Unique ID for the node
             input_schema=ReadDiscordMessagesBlock.Input,  # Assign input schema
             output_schema=ReadDiscordMessagesBlock.Output,  # Assign output schema
+            description="Reads messages from a Discord channel using a bot token.",
```

Comment:
is the bot token part required?

Reply:
ticket created
https://github.com/Significant-Gravitas/AutoGPT/issues/8170